src/account/KeylessAccount.ts

```diff
@@ -0,0 +1,371 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { JwtPayload, jwtDecode } from "jwt-decode";
+import EventEmitter from "eventemitter3";
+import { EphemeralCertificateVariant, HexInput, SigningScheme } from "../types";
+import { AccountAddress } from "../core/accountAddress";
+import {
+  AnyPublicKey,
+  AnySignature,
+  KeylessPublicKey,
+  KeylessSignature,
+  EphemeralCertificate,
+  Signature,
+  ZeroKnowledgeSig,
+  ZkProof,
+} from "../core/crypto";
+
+import { Account } from "./Account";
+import { EphemeralKeyPair } from "./EphemeralKeyPair";
+import { Hex } from "../core/hex";
+import { AccountAuthenticatorSingleKey } from "../transactions/authenticator/account";
+import { Deserializer, Serializable, Serializer } from "../bcs";
+import { deriveTransactionType } from "../transactions/transactionBuilder/signingMessage";
+import { AnyRawTransaction, AnyRawTransactionInstance } from "../transactions/types";
+import { AptsoDomainSeparator, CryptoHashable } from "../core/crypto/cryptoHasher";
+import { base64UrlDecode } from "../utils/helpers";
+
+export const IssuerToJwkEndpoint: Record<string, string> = {
```

Comment:
would we need to add to it as we have more providers?
lets add some string docs

Reply:
I removed this for now as I took out the functionality that utilized this.